azure-pipelines-pr.yml

```diff
@@ -0,0 +1,99 @@
+variables:
+# Cannot use key:value syntax in root defined variables
+- name: _PublishUsingPipelines
+  value: true
+- name: AzdoOrgUri
+  value: https://dev.azure.com/dnceng
+- name: AzdoProject
+  value: internal
+
+pr:
+  branches:
+    include:
+    - main
+    - production
+  paths:
+    include:
+    - '*'
+    exclude:
+    - '**/*.md'
+    - '.github/*'
+    - 'docs/*'
+    - CODE-OF-CONDUCT.md
+    - LICENSE.TXT
+    - README.md
+    - SECURITY.md
+    - THIRD-PARTY-NOTICES.TXT
+
+pool:
+  name: NetCore1ESPool-Internal-NoMSI
+  demands: ImageOverride -equals 1es-windows-2019
+
+stages:
+- ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+  - stage: GitHub_Issue_Verification
+    displayName: GitHub Issue Verification
+    dependsOn: []
+    # System.PullRequest.TargetBranch only evaulates during runtime conditions
+    condition: and(notIn(variables['System.PullRequest.TargetBranch'], 'refs/heads/production', 'production'), notIn(variables['System.PullRequest.SourceBranch'], 'refs/heads/production', 'production'))
```

Comment:
Should this be `main` and `production`?

Reply:
Not since these are different branch types (target and source). Its the same in the current version - https://github.com/dotnet/arcade-services/blob/ba165e6aa0bf16decf19e19292107458e12f89c3/azure-pipelines.yml#L51

Reply:
Ah, now I see the diff